engine/src/test/java/com/redhat/cloud/notifications/events/EndpointProcessorTest.java

```diff
@@ -161,6 +161,7 @@ void testTestEndpointAnsibleAliasToWebhook() {
         event.setOrgId(orgId);
 
         Mockito.when(this.endpointRepository.getTargetEndpoints(Mockito.anyString(), Mockito.any(EventType.class))).thenReturn(List.of(endpointFixture));
+        Mockito.when(this.endpointRepository.getTargetEndpointsWithoutUsingBgs(Mockito.anyString(), Mockito.any(EventType.class))).thenReturn(List.of(endpointFixture));
```

Comment:
Is this needed? Everything is disabled by default in the configuration, so `getTargetEndpointsWithoutUsingBgs` should not be called by this test.

Reply:
Now it is :)